### ISSUE TITLE
Version 3 onion services are unreachable

### ISSUE THREAD

**Issue by isuldor** [decred/dcrd#2035]
I believe dcrwallet already supports onion HiddenServiceVersion 3, because the address for CSPP is 6i6otcms7mxhnmqf5uclj6tot4gd6dsgk3twnfefssh7v7pbqs57umqd.onion.  So I had expected dcrd also supported HSv3.  I attempted to set up dcrd 1.5 in bridge mode, as described in [docs/configuring_tor.md](https://github.com/decred/dcrd/blob/master/docs/configuring_tor.md). But I got the error
```bash
[WRN] SRVR: Not adding 4mcqdwmx5b5dvfl6imumomqifypwctsgneppdslw7cjwvrvrogzigtad.onion as externalip: tor host is unreachable
```
I am able to workaround this issue by configuring HiddenServiceVersion 2 and using that instead.

**Comment by davecgh** [comment]
Correct.  They are not supported and doing so is a pretty hard change because the addr message format doesn't have enough space to support addresses that large.

**Comment by chappjc** [comment]
Can the issue remain open?

**Comment by dajohi** [comment]
https://github.com/rahra/onioncat/commit/a145dc176ef3758bfeb48f70d59a9c97f7819b1b

**Comment by dajohi** [comment]
https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki#Appendix_B_Tor_v3_address_encoding

**Comment by sefbkn** [comment]
Picking this one up.
